### DISCUSSION TITLE
Relationship on searchable select not working using getFormModel

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#5807] [Help] [answered]
Not sure if this is a bug - or I am doing something incorrect.

I have a component called "CreateShipment", where I have defined a select field like below:

```php
protected function getFormSchema(): array
{
    return [
                Select::make('consignee')->label('Consignee')->searchable()->relationship('consignee', 'name')
                ]),
    ];
}

protected function getFormModel(): string
{
    return Shipment::class;
}
```

Then on my `Shipment` model, I have this relationship defined:

```php
public function consignee()
{
    return $this->belongsTo(Consignee::class, 'consignee_id');
}
```

In my view, I get the following error:

> Livewire Entangle Error: Livewire property 'consignee' cannot be found

If I remove the `searchable()` method from the select field, it works without any issues.

**Comment by danharrin** [comment, marked as answer]
You need a `public $consignee` property as the error says.

Also, you need to call `$this->form->fill()` from `mount()`, always.

**Reply by oliverbj** [reply]
Thanks @danharrin. I thought it would be created by using `getFormModel`.

**Reply by danharrin** [reply]
`$this->form->fill()` is absolutely essential as well

**Reply by oliverbj** [reply]
Fixed it - thanks! And sorry, missed it in the documentation! :)
